pom.xml

```diff
@@ -61,6 +61,7 @@
         <module>providers/arangodb/shedlock-provider-arangodb</module>
         <module>providers/ignite/shedlock-provider-ignite</module>
         <module>providers/inmemory/shedlock-provider-inmemory</module>
+        <module>providers/memcached/shedlock-provider-memcached</module>
```

Comment:
I would prefer the mudule to be named `shedlock-provider-memcached-spymemcached` just in case somobody will want to provide another implementation using another library. It's not likely, but it might happen.

Reply:
YES